src/EditorFeatures/Core/Peek/PeekableItemSourceProvider.cs

```diff
@@ -20,15 +19,15 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Peek;
 [SupportsPeekRelationship("IsDefinedBy")]
 internal sealed class PeekableItemSourceProvider : IPeekableItemSourceProvider
 {
-    private readonly IPeekableItemFactory _peekableItemFactory;
+    private readonly PeekableItemFactory? _peekableItemFactory;
     private readonly IPeekResultFactory _peekResultFactory;
     private readonly IThreadingContext _threadingContext;
     private readonly IUIThreadOperationExecutor _uiThreadOperationExecutor;
 
     [ImportingConstructor]
     [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
     public PeekableItemSourceProvider(
-        IPeekableItemFactory peekableItemFactory,
+        [Import(AllowDefault = true)] PeekableItemFactory? peekableItemFactory,
```

Comment:
Why the AllowDefault=true here? I wouldn't expect this was optional anymore since you've got the type being exported in the same folder now.

Reply:
true.  will undo.